cves/CVE-2010-0643.yml

```
@@ -75,9 +87,11 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: Yes, there were automated unit tests involved in this vulnerability from the
```

For any multi-line comments, there it should look like below. Please fix this and other errors that the CI talks about. For another example, look at how the instructions do a multi-line answer.
```suggestion
  answer: |
   Yes, there were automated unit tests involved in this vulnerability from the
```